### PR TITLE
Gemini 2a infra

### PR DESCRIPTION
**Pull request by vedhavyas**
Add Gemini-2a infrastructure in DO
- Adds ~6~ 4 RPC nodes in APAC and NA each
- Adds 3 Bootstrap nodes in APAC and NA each

Note: I have removed gemini-1b as the infra is completely brought down outside this repo so syncing the state is not worth it

### CODE REVIEW

File: network-primitives/main.tf [hunk 1]
```diff
@@ -0,0 +1 @@
+/Users/vedhavyas/Projects/infra/resources/common.tf
```
Comment (nazar-pc) [inline network-primitives/main.tf]: Is this supposed to be committed?

**Review by nazar-pc — DISMISSED** [review]
Quite involved and I wasn't trying to analyze everything, a bit outside of my scope.
Generally makes sense. `network-primitives` is something I'm not sure we need, seems complex and there is just one specific comment.

File: network-primitives/main.tf (hunk 1 above)
Comment (vedhavyas) [reply network-primitives/main.tf]: Ahh no. The soft links :(
Will take it out

**Comment by vedhavyas** [comment]
Network primitives so that we can add another network with different config much much easily as most of the work is primitives. We can take it out if we don't use it for sure

**Review by nazar-pc — APPROVED** [review]

**Pull request merged**